service/internal/cloudconfig/template/instance_storage_class.go

```diff
@@ -4,8 +4,6 @@ const InstanceStorageClassContent = `apiVersion: storage.k8s.io/v1
 kind: StorageClass
 metadata:
   name: gp2
-  annotations:
-    storageclass.kubernetes.io/is-default-class: "true"
```

Comment:
this will be dropped because you can have only one storage class as default. the default one will come from EBS csi driver with a version `gp3`